activesupport/lib/active_support/duration.rb

```diff
@@ -235,6 +235,8 @@ def initialize(value, parts, variable = nil) # :nodoc:
     end
 
     # Returns a copy of the parts hash that defines the duration
+    #
+    #   1.years.parts # => {:years=>1}
```

Comment:
`1.years` isn't grammatically correct. Maybe another example makes it even clearer:

```suggestion
    # Returns a copy of the parts hash that defines the duration.
    #
    #   5.minutes.parts # => {:minutes=>5}
    #   3.years.parts # => {:years=>3}
```